client/src/modules/subsidies/subsidies.routes.js

```diff
@@ -1,9 +1,41 @@
 angular.module('bhima.routes')
-  .config(['$stateProvider', function ($stateProvider) {
+  .config(['$stateProvider', $stateProvider => {
     $stateProvider
       .state('subsidies', {
         url         : '/subsidies',
         controller  : 'SubsidyController as SubsidyCtrl',
         templateUrl : 'modules/subsidies/subsidies.html',
+      })
+
+      .state('subsidies.create', {
+        url : '/create',
+        params : {
+          creating : { value : true },
+        },
+        onEnter : ['$uibModal', subsidyModal],
+        onExit : ['$uibModalStack', closeModal],
+      })
+
+      .state('subsidies.edit', {
+        url : '/edit/:id',
```

Comment:
This is a small nitpick, but could you make this `:id/edit` or does that not work?